go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/ONSdigital/dp-frontend-router
+
+go 1.12
+
+require (
+	github.com/ONSdigital/dp-api-clients-go v1.0.0
```

Comment:
If you could update this to latest version of `dp-api-clients.go`; it should be on `v1.2.0`